docs/src/visualization.md

```diff
@@ -398,3 +398,59 @@ to convert all solution files. The default is to generate a PNG file for each
 variable found in the respective file. Use `format=:pdf` as a keyword argument
 to create PDF files. A comprehensive list of all possible arguments for
 `trixi2img` can be found in the [Trixi2Img.jl API](@ref).
+
+## [Makie.jl [experimental]](@id Makie.jl)
+
+In addition to [Plots.jl](@ref Plots.jl) support, Trixi includes visualization utilities through
+[Makie.jl](https://github.com/JuliaPlots/Makie.jl/). Trixi provides Makie-based visualization options
+both for heatmap-type plots (similar to the [Plots.jl](@ref Plots.jl) recipes) as well as for
+interactive surface plots. Support is currently limited to the [`UnstructuredMesh2D`](@ref) type.
+
+!!! note
+    Plotting via Makie.jl is still considered an experimental feature and might
+    change in any future releases.
+
+A Makie plot can be created as follows: after running a simulation with Trixi in the REPL, load a
+Makie backend (for example, [GLMakie](https://github.com/JuliaPlots/GLMakie.jl/) or
+[CairoMakie](https://github.com/JuliaPlots/CairoMakie.jl)).
+```julia
+julia> using GLMakie
+```
+
+To visualize the solution and mesh with a heatmap-type plot, simply run
+```julia
+julia> plot(sol)
+```
+!!! note
+    Both Makie.jl and Plots.jl export `plot`, so if you load both librarys, you will have to
```

Comment:
Wow, how did I miss this as a native English speaker?

Reply:
This is marked as resolved but not fixed in the sources. Did you, by any chance, hit `Add  suggestion to batch` and then either "Resolve conversation" or replied using the "Submit review" functionality? If you did, this will unremove the suggestion and not apply the change. Is there any chance this applies also to other comments I made?

Reply:
That's exactly what I did...

I'm checking over each comment now in case I missed something.